CHANGELOG.md

```diff
@@ -6,8 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [5.1.1](https://github.com/ASFHyP3/hyp3-gamma/compare/v5.0.1...v5.1.0)
 
-## [5.1.0](https://github.com/ASFHyP3/hyp3-gamma/compare/v5.0.1...v5.0.1)
+### Added
+* Ascending/descending and orbit number information have been added to the '*.txt' parameter file in InSAR products
+
+## [5.1.0](https://github.com/ASFHyP3/hyp3-gamma/compare/v5.0.1...v5.1.0)
 
 ### Added
```

Comment:
```suggestion
## [5.1.0](https://github.com/ASFHyP3/hyp3-gamma/compare/v5.0.1...v5.1.0)

### Added
* Ascending/descending and orbit number information have been added to the '*.txt' parameter file in InSAR products
```
Current released version is [v5.0.1](https://github.com/ASFHyP3/hyp3-gamma/releases) so you can add this to the v5.1.0 block -- I think we'll release them together.